cypress/support/step_definitions/testSetup.js

```diff
@@ -106,6 +109,14 @@ Given('the environment has been set up for {string} tests', (test) => {
       fireLog.fail('Marker creation failed');
     }
   }
+
+  if (test === 'Deeplink' && !Cypress.env('intentTemplates') && !Cypress.env('app_metadata')) {
+    cy.fetchAppMetaData().then((appMetaData) => {
+      Cypress.env('app_metadata', appMetaData);
+    });
+    const combinedIntentTemplates = _.merge(internalIntentTemplates, externalIntentTemplates);
+    Cypress.env('intentTemplates', combinedIntentTemplates);
+  }
```

Comment:
This cannot go in FCS. it has to be in config module I believe. Can be confirmed in calls

Reply:
moved to config module